### ISSUE TITLE
[EventCalendar] Quick Add places events on current date instead of target date

### ISSUE THREAD

**Issue by Zren** [Zren/plasma-applets#76]
Placing an event on Apr 25 with the text "Test" has an event created on Apr 20 (today) instead.

Our code is:

```
			var dateString = date.getFullYear() + '-' + (date.getMonth()+1) + '-' + date.getDate()
			var eventText = dateString + ' ' + text
			Shared.createGCalEvent({
				access_token: accessToken,
				calendarId: calendarId,
				text: eventText,
			}, function(err, data) {
```

Which places "2017-4-20 " (for the target date of April 20th) in front of the user entered text.

Changing my calendar to "sr" had no effect. I thought it meant cyrillic, but sr is Serbian not russian.

![](https://i.imgur.com/8ii9Zav.png)

Changing it to "ru"  Русский, it reproduces the bug.

**Comment by Zren** [comment]
Creating an event with "ru" using "test6" creates an event today, starting at 6:05 PM for 1 hour till 7:05 (that's probably when I made the event). Interesting that it doesn't make an all day event.

![](https://i.imgur.com/YOe0bem.png)

**Comment by shvchk** [comment]
Yeah, it seems Google just doesn't recognize dates on quick add when language is set to Russian (more languages are probably affected). I've tested lots of different types of dates adding events with [Calendar Chrome extension from Google](https://chrome.google.com/webstore/detail/google-calendar-by-google/gmbgaklkmjakoegficnlkhebmhkjfich), nothing worked, so for now I just changed language to English in Google Calendar settings.
